src/toil/wdl/wdltoil.py

```diff
@@ -3451,7 +3452,7 @@ def main() -> None:
                 # have to cast from more specific to less specific ones here.
                 # The miniwld values_from_json function can evaluate
                 # expressions in the inputs or something.
-                WDLTypeDeclBindings = WDL.Env.Bindings[Union[WDL.Tree.Decl, WDL.Type.Base]]
+                WDLTypeDeclBindings = WDL.Env.Bindings[WDL.Tree.Decl] | WDL.Env.Bindings[WDL.Type.Base]
```

Comment:
@DailyDreaming We can't actually use the `|` operator on types in actual code, or Python 3.8 and 3.9 can't execute the code.